modules/3d/include/opencv2/3d/ptcloud.hpp

```diff
@@ -8,6 +8,8 @@
 #ifndef OPENCV_3D_PTCLOUD_HPP
 #define OPENCV_3D_PTCLOUD_HPP
 
+#include "opencv2/flann.hpp"
```

Comment:
Through my attempts, the flann library does work independently of this part. Just need to store and pass the result of NN search to this part. But here is a question, knn search only supports CV_32F and CV_32S, is it possible to make the knn search in flann support CV_32F3 so that it can directly pass vector<Point3f> as parameter?

Reply:
Sorry, I forgot that 3d module already has `flann` in dependencies, my fault.
As of float3 data, I'm sure `flann` can process them if you `reshape` your data into 3 columns. I guess the reason is that they probably use something common enough to be put into L1/L2/LInf/... norm calculation function.
I think that kind of `reshape` can be done inside the `flann` code if that wouldn't break anything.

Reply:
Thanks! I'll try it.